### DISCUSSION TITLE
Daylight saving time in SOLWEIG

### DISCUSSION THREAD

**Discussion by j0lsson** [UMEP-dev/UMEP#471] [General]
Hi! 

Do Solweig account for daylight saving time? 

Thanks,

**Comment by biglimp** [comment]
No, the meteorological forcing data is local standard time.

**Reply by j0lsson** [reply]
Thanks for the quick reply! So if the time stamp of my observation are in "summer time", I need to offset it with 1h according to standard (winter) time? 

Best,

**Comment by biglimp** [comment]
Yes, correct.

**Reply by j0lsson** [reply]
Great, thanks for clarifying!

**Comment by ML426** [comment]
I have another question along the same lines,

When I use a weather data forcing with utc+1, should I also set the utc offset parameter to +1?

In my case, I get a very strange result compared to the measurement data. but using offset +0, it is close to the measurement.

Can you give me more information about this?

**Reply by biglimp** [reply]
Time in forcing data should be in local standard time and the utc set accordingly, e.g. sweden is utc=1.

**Reply by ML426** [reply]
Hi @biglimp ,

I come back to you to have more details on this topic.

If I have data in local time (in summer with the advance of one hour) I have to keep as such the parameter UTC offset at +1 ( e.g. if I am in sweden CET=UTC+1)
And on the other hand if my weather data is in UTC (without taking into account the daylight saving time) I should also keep the UTC offset = 1 or rather set it to zero?

Kind regards,

**Reply by biglimp** [reply]
I think you should set utc based on your weather file so =0.
The best way to test this is to put in a POI at an open location in your model (or at roof level, i.e. unshaded) and then compare sun altitude and Kdown. They should peak at the same time.
